content/en/agent/logs/proxy.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Agent proxy for logs
+title: TCP proxy for logs
```

Comment:
```suggestion
title: Proxy for logs
```

Reply:
So actually, I would like to have explicitly TCP in the title so that we have the main proxy documentation that is for HTTPS for all data type and this specific page if they are using TCP.

Reply:
👌 Sounds good @NBParis - sorry for the delay on this.